Smalltalk.gitignore

```diff
@@ -1,15 +1,24 @@
 # changes file
 *.changes
+*.chg
 
 # system image
 *.image
+*.img7
+*.img
 
 # Pharo Smalltalk Debug log file
 PharoDebug.log
 
 # Squeak Smalltalk Debug log file
 SqueakDebug.log
 
+# Dolphin Smalltalk source file
+*.sml
```

Comment:
The image of dolphin smalltalk 7 fresh is composed of 3 files, chg sml img7
The Dolphin Smalltalk 7 fresh image is composed of 3 files:
ImageName.chg
The Dolphin Smalltalk 7 fresh image has 3 file Dolphin.chg, Dolphin.sml and Dolphin.img7